problems/strings.js

```diff
@@ -117,7 +167,17 @@ function disemvowel() {}
  * example: disemvowelLongWords("hi i am your friend") returns "hi i am your frnd"
  * example: disemvowelLongWords("amber is the color of your energy") returns "mbr is the clr of your nrgy"
  */
- function disemvowelLongWords(){}
+ function disemvowelLongWords(str){
+   let splitAr = str.split(' ')
+  let newStr = ""
+  for (i = 0; i < splitAr.length;i++){
+    if(splitAr[i].length >= 5){
+      newStr += disemvowel(splitAr[i]) + ' '
+    }else{
+      newStr += splitAr[i]+ ' '
+    }
+  }return newStr.trim()
```

Comment:
This is great! No real feedback here. You're using another function that solves a portion of this problem, to help you solve this one!